src/Phansible/Provider/GithubProvider.php

```diff
@@ -15,15 +18,16 @@ class GithubProvider implements ServiceProviderInterface
      */
     public function register(Application $app)
```

Comment:
@debo I think we can now have return type on this

Reply:
They will come don't worry, I will go through all the inspections warning after I migrated the whole app to Symfony 4